sigma/processing/conditions/rule.py

```diff
@@ -174,7 +175,16 @@ def match(
 
         # Finally, value has some comparable type
         compare_value: Union[str, int, float, date, SigmaLevel, SigmaStatus]
-        if isinstance(value, (str, UUID)):  # exact match of strings and UUIDs
+        if isinstance(value, list):  # list membership check
+            if self.op == "in":
+                return self.value in value
+            elif self.op == "not_in":
+                return self.value not in value
+            else:
+                raise SigmaConfigurationError(
+                    f"Invalid operation '{self.op}' for list comparison in rule attribute condition {str(self)}. Only 'in' and 'not_in' are supported."
+                )
```

Comment:
The two operators "eq" and "ne" should work for lists too but always return False (for eq) and True (for ne).